src/class-parsely.php

```diff
@@ -1105,4 +1105,22 @@ public function get_api_key(): string {
 
 		return $this->api_key_is_set() ? $options['apikey'] : '';
 	}
+
+	/**
+	 * Get the default category name for posts.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @return string The category name.
+	 */
+	public function get_default_post_category_name(): string {
+		$category_id = get_option( 'default_category' );
+		$category    = get_categories( array( 'ID' => $category_id ) );
+
+		if ( 0 === count( $category ) ) {
+			return '';
+		}
+
+		return $category[0]->name;
+	}
```

Comment:
I think I would change this to `get_default_category()` and either return an array of:
 - slug
 - name
 - ID
 
 Or just return the object. That way, we know everything we need to know about the default category in one call.

```php
get_category( get_option( 'default_category', 1 )->slug;
```

If it's this simple, you may find that you don't even need a separate method to handle this, unless we want to mock it for unit tests.

Reply:
I've renamed the function according to your suggestion and made it to return the object or null.